src/providers/execution/contracts/oracle_daemon_config.py

```diff
@@ -0,0 +1,121 @@
+import logging
+from functools import lru_cache
+
+from web3 import Web3
+from web3.types import BlockIdentifier
+
+from src.providers.execution.base_interface import ContractInterface
+
+
+logger = logging.getLogger(__name__)
+
+
+class OracleDaemonConfigContract(ContractInterface):
+    abi_path = './assets/OracleDaemonConfig.json'
+
+    @lru_cache(maxsize=1)
+    def normalized_cl_reward_per_epoch(self, block_identifier: BlockIdentifier = 'latest') -> int:
+        response = self.functions.get('NORMALIZED_CL_REWARD_PER_EPOCH').call(block_identifier=block_identifier)
```

Comment:
Why other syntax?

Reply:
There is one function get that receives value by key.
To correctly work with cache and do not increase it's size, I think it's better to use different functions

Reply:
Didn't get it, to be honest.

Reply:
> @madlabman function `get` is the contract function that returns value by key. `NORMALIZED_CL_REWARD_PER_EPOCH` isn't state variable

Got it!

Reply:
fixed